cmp/tab/TabModel.ts

```diff
@@ -38,7 +39,7 @@ export interface TabConfig {
     icon?: ReactElement;
 
     /** Tooltip for the Tab in the container's TabSwitcher. */
-    tooltip?: ReactNode;
+    tooltip?: JSX.Element | string;
```

Comment:
This is new to me - especially vs. `ReactElement` - would like to understand more

Reply:
This change was to conform with Blueprint's accepted prop type (ReactNode was too broad)